tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -452,6 +457,18 @@ public function pushToLastStatusCodesArrays():void {
 		}
 	}
 
+	/**
+	 * @param string $emailAddress
+	 *
+	 * @return void
+	 */
+	public function pushEmailRecipientAsMailBox(string $emailAddress):void {
+		$splitEmailForUserMailBox =  explode("@", $emailAddress);
+		if (!\in_array($splitEmailForUserMailBox[0], $this->emailRecipients)) {
+			$this->emailRecipients[] = $splitEmailForUserMailBox[0];
+		}
```

Comment:
does this look cleaner?
```suggestion
		$mailBox =  explode("@", $emailAddress)[0];
		if (!\in_array($mailBox, $this->emailRecipients)) {
			$this->emailRecipients[] = $mailBox;
		}
```

Reply:
yes thanks for reporting !! :+1: